### PR TITLE
Sm/installer promotion fix

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
resolves the issue that was causing installers to point at old versions 

### What issues does this PR fix or reference?
@W-0@

### CODE REVIEW

**Review by RodEsp — COMMENTED** [review]
I prefer having the separate jobs for packaging/uploading and promoting. Why consolidate them into one?

Isn't the `checkout` step pulling the latest commit on the main branch so it should be the same across jobs in a workflow?

**Review by amphro — APPROVED** [review]

**Comment by mshanemc** [comment]
> I prefer having the separate jobs for packaging/uploading and promoting. Why consolidate them into one?

Agreed, but it wasn't working (the oclif `promote` command is uploading targeted tarballs anyway and there wasn't a good way to prevent it from doing that without making bigger changes to oclif). 

> Isn't the `checkout` step pulling the latest commit on the main branch so it should be the same across jobs in a workflow?  
Yes...any reason you're asking?

**Comment by RodEsp** [comment]
Never mind, I thought whatever was checked out had something to do with what the `promote` command did but seems like it doesn't.

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,36 +93,6 @@ jobs:
           echo $WINDOWS_SIGNING_KEY | base64 --decode > /tmp/windows-signing.pfx
       - run: yarn pack:win
       - run: yarn upload:win
```
Comment (RodEsp) [inline .circleci/config.yml:95]: Should we be running the `promote-stable-rc-win` script here or just not yet?

**Pull request merged**